tests/integrations/python/tests/test_google.py

```
@@ -2732,6 +2734,197 @@ def test_40b_input_tokens_long_text(self, google_client, test_config, provider,
             f"Long text should have >100 tokens, got {response.total_tokens}"
         )
 
+    # =========================================================================
+    # GOOGLE SEARCH GROUNDING TEST CASES
+    # =========================================================================
+
+    @skip_if_no_api_key("google")
+    def test_42_google_search_grounding(self, test_config):
```

Ruff (0.14.13)

Unused method argument: `test_config`

(ARG002)